Example/Emission/AppDelegate.m

```diff
@@ -181,6 +181,7 @@ - (void)setupEmissionWithUserID:(NSString *)userID accessToken:(NSString *)acces
     }
     UIViewController *viewController = [self viewControllerForRoute:route];
     UINavigationController *navigationController = [[EigenLikeNavigationController alloc] initWithRootViewController:viewController];
+    navigationController.navigationBarHidden = NO;
```

Comment:
Since the nav bar was always hidden, the left bar button item set on the next line was never visible.